datadog/resource_datadog_dashboard.go

```diff
@@ -1173,6 +1186,9 @@ func buildTerraformWidget(datadogWidget datadogV1.Widget, k *utils.ResourceDataK
 	} else if widgetDefinition.SLOWidgetDefinition != nil {
 		terraformDefinition := buildTerraformServiceLevelObjectiveDefinition(*widgetDefinition.SLOWidgetDefinition)
 		terraformWidget["service_level_objective_definition"] = []map[string]interface{}{terraformDefinition}
+	} else if widgetDefinition.SLOListWidgetDefinition != nil {
+		terraformDefinition := buildTerraformSloListDefinition(*widgetDefinition.SLOListWidgetDefinition, k.Add("slo_list_definition.0"))
```

Comment:
can we unify using either `SLO` or `Slo`? seems makes more sense to SLO as it is used in all other places?

Reply:
Yeah, this is definitely something I grappled with when writing the code. I ultimately decided to go with the "true" camelCase style after seeing that's how RUM and APM are referred to in the codebase (at least in the small sample size I looked at), for example:

https://github.com/DataDog/terraform-provider-datadog/blob/9ffc78a61b7b61f61d3e4c8f0d73ba36e2ad3772/datadog/resource_datadog_dashboard.go#L7705

https://github.com/DataDog/terraform-provider-datadog/blob/9ffc78a61b7b61f61d3e4c8f0d73ba36e2ad3772/datadog/resource_datadog_dashboard.go#L7826

For SLOs specifically, I see usage of both `"SLO"` and `"Slo"` throughout the codebase, so we're definitely inconsistent there

Reply:
I would be happy to switch over to the all-caps version though if that's preferred

Reply:
Agreed that the naming is a bit inconsistent everywhere in this resource. For the purposes of this PR, I am okay with keeping it as is (since it correlates with existing naming convention already used with rum, apm, etc). However, I will create a card on our backlog to make it consistent everywhere since I'm sure this spans across resources and is not just limited to this case.

Reply:
agreed and thanks @skarimo for taking care of it!